packages/stryker/src/initializer/StrykerInitializer.ts

```diff
@@ -23,12 +23,16 @@ export default class StrykerInitializer {
     this.patchProxies();
     const selectedTestRunner = await this.selectTestRunner();
     const selectedTestFramework = selectedTestRunner ? await this.selectTestFramework(selectedTestRunner) : null;
+    const selectedMutator = await this.selectMutator();
+    const selectedTranspilers = await this.selectTranspilers();
     const selectedReporters = await this.selectReporters();
-    const npmDependencies = this.getSelectedNpmDependencies([selectedTestRunner, selectedTestFramework].concat(selectedReporters));
+    const npmDependencies = this.getSelectedNpmDependencies([selectedTestRunner, selectedTestFramework, selectedMutator].concat(selectedTranspilers).concat(selectedReporters));
```

Comment:
This is a really long line of code. Maybe we should align on some max length. It took me a while to see that `transpilers` were included in this list.